### PR TITLE
Update OCI Discover Extension Response

### PR DESCRIPTION
**Pull request by akashsinghal**
- Updates the oci discover extension sample response to include the `endpoints` field and updated `url`

### CODE REVIEW

File: manifest-referrers-api.md [hunk 1]
```diff
@@ -34,8 +34,11 @@ Content-Type: application/json
     "extensions": [
         {
             "name": "cncf.oras.referrers",
+            "url": "https://github.com/opencontainers/distribution-spec/blob/main/extensions/_oci.md",
```
Comment (sajayantony) [inline manifest-referrers-api.md]: The value for url description should be this this repo. You can just use the repo root. 
Actually it’s https://oras.land

Comment (akashsinghal) [reply manifest-referrers-api.md]: ok I'll switch it to https://oras.land

Comment (SteveLasker) [reply manifest-referrers-api.md]: > URL link to the documentation defining this extension and its endpoints.

Shouldn't this be the url of the spec? https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md

**Review by SteveLasker — APPROVED** [review]
LGTM

**Comment by sajayantony** [comment]
Recommend fixing this and backporting to RC1 branch since this makes sure that clients can use the correct property to get the referrers URL.

File: manifest-referrers-api.md (hunk 1 above)
Comment (sajayantony) [reply manifest-referrers-api.md]: The value is currently only a place holder and the definition in the extension spec is 
```
url string, REQUIRED

URL link to the documentation defining this extension and its endpoints.
```

I think cloud providers can point to their own docs location.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by sajayantony — APPROVED** [review]

**Review by michaelb990 — APPROVED** [review]

**Pull request merged**